mongo/options/findoptions.go

```diff
@@ -209,6 +215,8 @@ func (f *FindOptions) SetSkip(i int64) *FindOptions {
 }
 
 // SetSnapshot sets the value for the Snapshot field.
+//
+// Deprecated: This option has been deprecated in MongoDB version 4.0. The default value is false.
```

Comment:
```suggestion
// Deprecated: This option has been deprecated in MongoDB version 3.6. The default value is false.
```

Reply:
I wanted to keep the notice the same for the struct field and the setter so I made them both
```
// Deprecated: This option has been deprecated in MongoDB version 3.6 and removed in MongoDB version 4.0.
```